nistats/first_level_model.py

```diff
@@ -583,6 +585,94 @@ def compute_contrast(self, contrast_def, stat_type=None,
 
         return outputs if output_type == 'all' else output
 
+    def get_voxelwise_model_attribute_(self, attribute, timeseries=True):
+        """Transform RegressionResults instances within a dictionary
+        (whose keys represent the autoregressive coefficient under the 'ar1'
+        noise model or only 0.0 under 'ols' noise_model and values are the
+        RegressionResults instances) into input nifti space.
+
+        Parameters
+        ----------
+        attribute : str
+            an attribute of a RegressionResults instance
+        timeseries : bool, optional
```

Comment:
could it be: 'result_as_time_series' ? This would be clearer I think.

Reply:
I agree, changed  `timeseries` to `result_as_time_series`